webroot/adm/keysets.html

```diff
@@ -26,6 +26,8 @@ <h3>Inputs</h3>
   <input type="text" id="addSiteIds" name="addSiteIds">
   <label for="removeSiteIds">Remove Allowed Sites:</label>
   <input type="text" id="removeSiteIds" name="removeSiteIds">
+  <label for="nullify">Set allowed_sites to null:</label>
+  <input type="checkbox" id="nullify" name="nullify" value="true">
 </div>
 
 <br>
```

Comment:
what does "Reset Keyset" do?

Reply:
This removes all sites from the allowlist to make it [].